tests/php/ModelAdminTest.php

```diff
@@ -209,30 +207,8 @@ public function testGetManagedModels()
 
     public function testGetManagedModelTabs()
     {
-        $mock = $this->getMockBuilder(ModelAdminTest\MultiModelAdmin::class)
-            ->setMethods(['getManagedModels'])
-            ->getMock();
-
-        // `getManagedModelTabs` relies on `getManagedModels` whose output format has changed within the 4.x line.
-        // We need to mock `getManagedModels` so it returns both the legacy and updated format.
-        $mock->expects($this->atLeastOnce())
-            ->method('getManagedModels')
-            ->will($this->returnValue([
-                'Player' => [
-                    'dataClass' => Player::class,
-                    'title' => 'Ice Hockey Players'
-                ],
-                Player::class => [
-                    'title' => 'Rugby Players'
-                ],
-                'cricket-players' => [
-                    'dataClass' => Player::class,
-                    'title' => 'Cricket Players',
-                ],
-            ]));
-
-
-        $tabs = $mock->getManagedModelTabs()->toNestedArray();
+        $admin = new ModelAdminTest\MultiModelAdmin();
+        $tabs = $admin->getManagedModelTabs()->toNestedArray();
```

Comment:
Had to remove this mock to avoid the following error:
> SilverStripe\Admin\Tests\ModelAdminTest::testGetManagedModelTabs
PHPUnit\Framework\MockObject\BadMethodCallException: Static method "getManagedModels" cannot be invoked on mock object

Thankfully this mock was only here to support a legacy format we don't need to support anymore - and this change passes without changing what is actually being tested anyway.